feature2/another-feature.txt

```diff
@@ -0,0 +1,3 @@
+feature2 branch 
+
```

Comment:
Is this space necessary?

Reply:
Yes this space is necessary for readability reason.